landscape-report.md

```diff
@@ -0,0 +1,418 @@
+---
+title: "Sharing Government Software"
+---
```

Comment:
thoughts on where this should live in the site? we don't have to link there yet but want to get the convo started

Reply:
Probably a section along the left called "Reports", with this and the other reports that are up at the Beeck site?